server/resp/src/main/java/org/infinispan/server/resp/Consumers.java

```diff
@@ -54,4 +55,36 @@ private Consumers() {
 
       GET_BICONSUMER.accept(null, alloc);
    };
+   public static final BiConsumer<LCSResponse, ByteBufPool> LCS_BICONSUMER = (res, alloc) -> {
+      // If lcs present, return a bulk_string
+      if (res.lcs != null) {
+         Resp3Handler.handleBulkResult(new String(res.lcs, StandardCharsets.US_ASCII), alloc);
```

Comment:
This is very wasteful to convert this to a String when it is already a byte[]. We really need a Resp3Handler method that takes a byte[] that does no String creation. 

It can just invoke ByteBufferUtils.bytesToResult that does this. It can be almost twice as fast.

Reply:
changed to avoid strings